### ISSUE TITLE
[Feature request]: Generate nullable properties as required

### ISSUE THREAD

**Issue by Eirenarch** [domaindrivendev/Swashbuckle.AspNetCore#2969]
### Is your feature request related to a specific problem? Or an existing feature?

When generating OpenAPI json in most cases it is desirable that non-nullable properties are marked as required. As far as I can tell currently this can be achieved by adding a filter like the one described [here](https://stackoverflow.com/a/68987970). While this is not very hard I think that this feature is desirable for the vast majority of projects so it would be good if it is available out of the box.

### Describe the solution you'd like

Maybe a parameter on the SupportNonNullableReferenceTypes method to enable this behavior

**Comment by billhong-just** [comment]
@Eirenarch https://github.com/domaindrivendev/Swashbuckle.AspNetCore/pull/2803

**Comment by Eirenarch** [comment]
> @Eirenarch Maybe this [PR](https://github.com/domaindrivendev/Swashbuckle.AspNetCore/pull/2803) meets your need.

Looks like it. Has it shipped yet?

**Comment by martincostello** [comment]
Yes - if you look at the tags on the merged commit you can see the tags for the releases that contain it: https://github.com/domaindrivendev/Swashbuckle.AspNetCore/commit/85fefe4257f718ae669b67f0b562177ce25f7934

**Comment by jgarciadelanoceda** [comment]
@martincostello I think this could be a feature that was added in #2710. If you think the same I can create other PR to make this new MarkNonNullableReferenceTypesAsRequired

**Comment by jgarciadelanoceda** [comment]
@martincostello this can be closed.. You have the required keyword that works as expected (If you mark this over a property makes the property required in the OpenApi).
And the other way, when you want to make all the properties that are not nullable as required with the NonNullableReferenceTypesAsRequired, I think that is more than covered with these two options.
